.github/workflows/backmerge.yml

```diff
@@ -0,0 +1,24 @@
+name: Backmerge main -> next
+
+on:
+    push:
+        branches:
+            - 'main'
```

Comment:
Maybe we can use the [release](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) event?

Reply:
LGTM! We can avoid useless run!

Reply:
I was thinking, we have to be able to separate `releases` from `prereleases`, it should be handled by the `release` and `pre-release` event but I am not sure 🤔

Reply:
I think we need to add a check on the type `released` in the job

Reply:
I think the `released` action type won't get `pre-release` they also have a different [tag](https://github.com/immobiliare/fastify-metrics/releases/tag/v2.0.0-next.3) looking at documentation seems like to me that if you want both regardless of type you should use `published`, at least that's my understanding